.github/workflows/canary-test-config/action.yaml

```
@@ -16,7 +16,7 @@ runs:
     - name: Setup Minikube
       shell: bash --noprofile --norc -eo pipefail -x {0}
       run: |
-        tests/scripts/github-action-helper.sh install_minikube_with_none_driver v1.27.1
+        tests/scripts/github-action-helper.sh install_minikube_with_none_driver v1.27.2
```

```suggestion
        tests/scripts/github-action-helper.sh install_minikube_with_none_driver v1.27.3
```
v1.27.3 is the latest Kubernetes release, let's move to that.

---

Done.